### PR TITLE
Tom UI merge test

### PR DESCRIPTION
**Pull request by TomxHWRxWI20C**
all test are invalid or missing
but the rest should be awesome

### CODE REVIEW

File: models/CalculationModel.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class CalculationModel {
+	private String selectedDay;
+	private String workBegin;
+	private String workEnd;
+	private String totalWorkTime;
+	private String totalBreakTime;
+
```
Comment (valientes) [inline models/CalculationModel.java:15]: The parts that are there are fine to me. I'm just missing overtime/flexiTime

**Review by valientes — APPROVED** [review]
lgtm. I forgot our agreement regarding flexiTime, but I think it was fine, so you got my approval

File: controller/LoginController.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    private TextField userinput;
+
+    @FXML
+    private PasswordField passswordinput;
+
+    @FXML
+    private Button loginbutton;
+
+    @FXML
+    private MenuItem gottoHelpCenter1;
```
Comment (jerdbaer) [inline controller/LoginController.java:27]: what is this?

File: controller/PopupLimitsController.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	@FXML
+    private TextField txtFieldReason;
+
+    @FXML
+    private Label txtError;
+
+    
+    @FXML
+    public void initialize() {
+    	var calculations = MainController.getCalculationModel();
```
Comment (jerdbaer) [inline controller/PopupLimitsController.java:21]: MainController.getCalculationModel();

Comment (jerdbaer) [reply controller/PopupLimitsController.java:21]: got it, weil du es immer als gleiche Codezeile benutzt ;) 
könnte man hier jedoch trotzdem wegnehmen

File: models/CalculationModel.java (hunk 1 above)
Comment (jerdbaer) [reply models/CalculationModel.java:15]: @valientes  will be added when we get the plannedWorkingTime from Database :)

**Review by jerdbaer — APPROVED** [review]
see comments, just questions no code failure

File: models/CalculationModel.java (hunk 1 above)
Comment (valientes) [reply models/CalculationModel.java:15]: @jerdbaer ah, I remember now we talked about that earlier today. My memory is a mess rn...

File: controller/LoginController.java (hunk 2 above)
Comment (TomxHWRxWI20C) [reply controller/LoginController.java:27]: an ancient relic from the time bevor @TomxHWRxWI20C  toke over
can be deleted

**Pull request merged**